gcc/rust/expand/rust-macro-expand.cc

```diff
@@ -3735,10 +3763,10 @@ MacroExpander::transcribe_rule (
     = substitute_tokens (invoc_stream, macro_rule_tokens, matched_fragments);
 
   // // handy for debugging
-  // for (auto &tok : substituted_tokens)
-  //   {
-  //     rust_debug ("tok: [%s]", tok->as_string ().c_str ());
-  //   }
+  for (auto &tok : substituted_tokens)
```

Comment:
we should probably make a helper at some point to turn this into a simple std::string at some point in another PR